### DISCUSSION TITLE
Jekyll CSS not working

### DISCUSSION THREAD

**Discussion by Yuki9371** [community/community#140284] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, I am a beginner trying to build a jekyll website and publish on GitHub pages. It seems that the CSS is not working on my website. I looked for similar discussions and tried some solutions like changing the position of the CSS file and editing the path of CSS, but it was still not working. I am not sure what's wrong with it. 
I would appreciate any suggestions.
This is my repo: https://github.com/Yuki9371/yuki.github.io
![pic2](https://github.com/user-attachments/assets/1bee0eee-1955-418b-908d-35ed16d8a010)
![pic](https://github.com/user-attachments/assets/2a590ca7-6cd0-453b-b711-4b46c3bdf5dd)

**Comment by davevad93** [comment, marked as answer]
Hi @Yuki9371 ,

I took a look at your repo, specifically in your `index.html`, remove the forward slash in your css path, change that line to this: 

```html
<link rel="stylesheet" href="assets/main.css">
```

Result:

![1](https://github.com/user-attachments/assets/ee567741-9612-4ccb-a424-0dcdbb46136a)

**Comment by Yuki9371** [comment]
Hi David,

Thank you so much for your help. It works really well right now. I have
closed the discussion so I just sent this message through email.

Thank you,
Yuki

**Reply by davevad93** [reply]
You're welcome @Yuki9371 , have a nice day! 👍
